packages/data-bite/src/CdcDataBite.tsx

```diff
@@ -0,0 +1,297 @@
+import React, { useEffect, useRef, useState } from 'react';
+import EditorPanel from './components/EditorPanel';
+import defaults from './data/initial-state';
+import Loading from '@cdc/core/components/Loading';
+import ResizeObserver from 'resize-observer-polyfill';
+import Context from './context';
+// @ts-ignore
+import CircleCallout from './components/CircleCallout';
+import './scss/main.scss';
+import * as Constants from './constants/index';
```

Comment:
Maybe name constants you're using? ex `import { NAME, NAME_2 } from './constants/index';` and maybe consider moving this to the CdcDataBite file instead of giving it its own folder

Reply:
Removed constants file and added to CdcDataBite.tsx